images/scripts/fedora.setup

```diff
@@ -200,6 +202,11 @@ fi
 # Configure kubernetes
 [ -z "$HAVE_KUBERNETES" ] || /var/lib/testvm/kubernetes.setup
 
+# for Fedora ≥ 31, use podman for cockpit/base container; we don't need kubernetes any more
```

Comment:
maybe stupid question, but what do we need `cockpit/base` for except for bastion testing?

Reply:
We also need it for testing cockpit/kubernetes on the rhel-7.8 branch. But on master, that's the remaining use case, yes. We could probably get rid of it and use the same approach as for cockpit/ws -- but that is based on the official container image, which doesn't exist yet for bastion.